java-lib/src/main/java/org/clink/util/ClinkJNA.java

```diff
@@ -12,27 +14,26 @@
  * limitations under the License.
  */
 
-package org.clink.example;
+package org.clink.util;
 
 import com.sun.jna.Library;
 import com.sun.jna.Native;
 import com.sun.jna.Pointer;
+import org.clink.linalg.SparseVectorJNA;
 
-import java.util.Properties;
+/** Utility methods that have implementations in C++. */
+public interface ClinkJNA extends Library {
+    ClinkJNA INSTANCE = Native.load("clink_jna", ClinkJNA.class);
 
-/** Simple example of native library declaration and usage. */
-public class Main {
+    double SquareAdd(double x, double y);
 
-    public interface ClinkKernels extends Library {
-        ClinkKernels INSTANCE = Native.load("clink_kernels_jna", ClinkKernels.class);
+    double Square(double x);
 
-        double SquareAdd(double x, double y);
+    Pointer OneHotEncoderModel_loadFromMemory(
```

Comment:
Could we have comments for these methods so that other developers can follow in the future?

Ideally the comments could make it clear that these three methods belong to the same `OneHotEncoder` group.

Reply:
OK. I'll try to add proper comments.